server/api/artists.js

```diff
@@ -1,5 +1,5 @@
 const artistRouter = require('express').Router();
-// const {Artist} = require('../db/models/products/...')  // enter the correct model name and address here
+// const { Artist } = require('../db/models/products/artist'); // enter the correct model name and address here
```

Comment:
1. If we're pulling in Artist from this file, it should not be destructured because it's not being exported as an object in the /models/products/artist.
2. However, moving forward with all of our routes, I think we should be pulling in our models from the /models/index file, which will simplify all of our imports in our routes -- and _then_ we'll need to destructure them.

Reply:
Thanks Emily for catching that. I have changed it and it should be g2g now